Add partner docs templates to non-build folder (#4145)
segmentio/segment-docs: templates/partners/destination-new.md

{% include content/plan-grid.md name="actions" %}

<!-- Include a brief description of the destination here, along with a link to your website. -->

<!-- This include describes the requirement of A.js 2.0 or higher for Actions compatibility, and is required if your destination has a web component. -->

{% include content/ajs-upgrade.md %}

<!-- In the section below, explain the value of this actions-based destination. If you don't have a classic version of the destination, remove this section. -->

## Benefits of <destination_name> (Actions)

<destination_name> (Actions) provides the following benefits:

- **Main point 1**. One or two sentences that back up the main point.
- **Main point 2**. One or two sentences that back up the main point.

<!-- The section below explains how to enable and configure the destination. Include any configuration steps not captured below. For example, obtaining an API key from your platform and any configuration steps required to connect to the destination. -->

## Getting started

> 1. From the Segment web app, click **Catalog**, then click **Destinations**.

Reviewer: ```suggestion
1. From the Segment web app, navigate to **Connections > Catalog** and select the **Destinations** tab within the catalog. 
```

> 2. Find the Destinations Actions item in the left navigation, and click it.

Reviewer: ```suggestion
2. Search for **<destination_name>** and select the destination.
```

> 3. Click **Configure <desintation_name>**.

Author: ```suggestion
3. Click **Configure <destintation_name>**.
```

> 4. Select an existing Source to connect to <destination_name> (Actions).

Reviewer: ```suggestion
4. Select the Source to connect to <destination_name> (Actions).
```
